docs/analysis.md

```diff
@@ -503,6 +503,8 @@ Here are the exceptions and their meanings:
 
 - `RuleEvaluationConditionMet`: This is raised when the rule invocation returns True for some step.
 
+- `MissingCollectionFiles`: This is raised when the training job has ended but all the collection files could not be loaded by the trial.
```

Comment:
Can we also add something like: This means that no data was saved for your training job. Please check your Hook configuration and whether your training job is supported from the relevant framework description page.